backend/src/layer/data/database/request.rs

```diff
@@ -11,13 +11,12 @@ use crate::{
     util::{Additive, Allergen, Date, MealType, Price, Uuid},
 };
 
-/// We have for four weeks, including the current week
-const MAX_WEEKS_DATA: i64 = 4;
-
 /// Class implementing all database requests arising from graphql manipulations.
 #[derive(Debug)]
 pub struct PersistentRequestData {
     pub(super) pool: Pool<Postgres>,
+    /// Number of weeks, including the current week, we get/have data for.
+    pub(super) max_weeks_data: u32,
```

Comment:
Wieso ist das pub(super)?

Reply:
because only the Factory is allowed to access these attributes, but the Factoriey lies in another file